ui/desktop/app/controllers/application.js

```diff
@@ -75,4 +102,34 @@ export default class ApplicationController extends Controller {
         rootEl.classList.remove('rose-theme-light');
     }
   }
+
+  @action
+  signoutAttempt() {
+    this.showSignoutModal = true;
```

Comment:
I don't see any logic in when this should be activated, it looks like this popup will always appear when trying to logout even if there are no active sessions. If we remove `setSignoutInProgress`, we shouldn't need this `signoutAttempt` action anymore and just do everything in `invalidateSession`